polymer.json

```diff
@@ -0,0 +1,5 @@
+{
+  "lint": {
+    "ignoreWarnings": ["overriding-private"]
```

Comment:
I had to add this because of overriding `_openedChanged` property observer, which is marked as `@private` in `vaadin-context-menu`. We probably don't want to expose our ugly workarounds publicly 🙈

Reply:
So should we change it in `vaadin-context-menu` to be `@protected`? Or would it be better to create an issue in context-menu about refactoring it somehow so that we can do what we need without overriding private (might be new feature)?

Reply:
Ideally we should have a common primitive. Submitted https://github.com/vaadin/vaadin-core/issues/247